src/pages/index.tsx

```diff
@@ -59,17 +59,21 @@ const Home = () => {
   };
 
   return (
-    <>
-      <WelcomeCard isLoggedIn={!!currentUser} />
+    !feedRes.loading && (
```

Comment:
What is this conditional achieving for us?

Reply:
Nothing, that was part of a previous test and I must have forgotten to remove it. I will correct this in the next commit.